Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -13009,7 +13009,7 @@
         <ScoreMake value="250"/>
         <ScoreKill value="250"/>
         <ScoreResult value="BuildOrder"/>
-        <SubgroupPriority value="20"/>
+        <SubgroupPriority value="25"/>

```

Comment:
DT - Autocast, Archon merge and Blink on upgrade - should be higher than Avenger